app/views/casa_cases/_thank_you_modal.html.erb

```diff
@@ -2,9 +2,18 @@
   <div class="modal fade" id="thank_you" data-backdrop="static" data-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
     <div class="modal-dialog modal-dialog-centered">
       <div class="modal-content">
-        <div class="modal-body">
+        <div class="modal-header">
           <%= "Case contact was successfully created. #{thank_you_message}" %>
         </div>
+        <% if when_do_we_have_court_dates(@casa_case) == 'future' %>
+          <div id="prompt-update-user" class="modal-body">
+            Please enter the previous court date if you know it so that court reports can be generated with the correct case contacts.
+          </div>
+        <% elsif when_do_we_have_court_dates(@casa_case) == 'past' %>
```

Comment:
**suggestion:** store `when_do_we_have_court_dates(@casa_case)` in a variable to avoid querying the database multiple times at different points in time.